packages/front-end/components/Radix/Dropdown.tsx

```diff
@@ -0,0 +1,78 @@
+// radix dropdown wrapper
+import { DropdownMenu as RadixDropdownMenu } from "@radix-ui/themes";
+// import colors from radix-ui themes
+import type { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+type AllowedChildren = React.ReactNode;
+
+type Props = {
+  trigger: React.ReactNode;
+  disabled?: boolean;
+  errorLevel?: "error" | "warning";
+  description?: string;
+  children: AllowedChildren;
```

Comment:
we should have a color prop here for changing the color on hover for `DropdownMenu.Content`. Looking at the design I think we only want to allow red and violet

Reply:
color is already on the items them self